app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -441,11 +505,17 @@ private void displayCurrentInfoFromReading(BgReading lastBgReading, boolean pred
             }
         }
         int minutes = (int)(System.currentTimeMillis() - lastBgReading.timestamp) / (60 * 1000);
-        notificationText.append("\n" + minutes + ((minutes==1)?" Minute ago":" Minutes ago"));
+        String minutesString;
+        if(BgGraphBuilder.isXLargeTablet(getApplicationContext()) || BgGraphBuilder.isLargeTablet(getApplicationContext())) {
+            minutesString = " Min ago";
```

Comment:
It is needed. Making this a separate field did not change anything from the way that things look.